src/lab_sim/package.xml

```diff
@@ -14,6 +14,8 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
+  <build_depend>ros-humble-examples-rclcpp-minimal-service</build_depend>
```

Comment:
I don't think this is included in https://index.ros.org/p/ros2cli_common_extensions/#humble-deps so we still need this